packages/components/src/apollo/EventsFederationApolloClient.ts

```diff
@@ -43,6 +43,14 @@ type GraphQLErrorsExtensionResponse = {
   status: number;
 };
 
+/**
+ * The Events-graphql-proxy offers a way to ignore some custom ignorable graphql errors.
+ * The 'X-Ignored-Error-Code' header is used to list the errors wanted to be ignored.
+ * */
+export const ignoredErrorCodesHeader = {
+  'X-Ignored-Error-Code': 'UNPOPULATED_MANDATORY_DATA',
+};
```

Comment:
These should be moved to some constants file and they should be given by the app through the apollo configuration property from appConfig file.